### ISSUE TITLE
mc-restic-compose-backup container not usable

### ISSUE THREAD

**Issue by Luctastic** [sVoxelDev/minecraft-server-template#11]
Hello :) 
Thanks for this awesome template, but I got some problems regarding your "mc-restic-compose-backup" image in combination with this template.

[Pastebin #1](https://pastebin.com/XznJADA1)
[Pastebin #2](https://pastebin.com/RCWe0dx0)

1) Cloned template and started via ./dc.sh up -d
2) Attached to backup-container via ./backup.sh attach
3) rcb status

First problem (see pastebin #1):

- Port var is null in python
- Tries to get it from container with tag restic-compose-backup.minecraft: true
- Workaround: Passing environment var RCON_PORT into minecraft container

```
environment:
      ...
      # used to pass port to backup
      RCON_PORT: ${RCON_PORT}
```

--------

Second problem (see pastebin #1):

- Can't connect to database
- Hardcoded root username in mc-restic-compose-backup image
- Searches for MYSQL_ROOT_PASSWORD env var in database container
- Workaround: Passing environment var MYSQL_ROOT_PASSWORD into database container (other problems occur now, due to the MYSQL_RANDOM_ROOT_PASSWORD env set)

--------

Third problem: (after temp-fixing first and second problem) (see pastebin #2)

- Can't set CRON_SCHEDULE environment var
  - If set, I get the error "exit status 2 from user root source /env.sh && rcb backup > /proc/1/fd/1"
  - Running source /env.sh inside the backup container, returns the following error message
    - "sh: export: line 20: entrypoint.sh: bad variable name"
  - In line 20 is my environment var CRON_SCHEDULE
- Workaround: Removing CRON_SCHEDULE from backup.env - default cronjob gets executed now, but I can't adjust the timings

**Comment by Silthus** [comment]
Hey @Luctastic 

did you execute the `init.sh` script after cloning the template? It sounds like a bunch of environment variables are missing which are set during the init phase.

**Comment by Luctastic** [comment]
Yeah sorry, I did execute the init script, forgot to mention that (double checked now)

1) Cloned template
2) sudo ./init.sh

```
Empty plugin URL
(Bungeecord) Port [25565]:
Query Port [25566]:
RAM for the Server [4G]:
base url of this environment, e.g.: test.your-server.net? [your-server.net]: dev.testing.eu
environment of this server (e.g.: test, prod)? [test]
Entered restic password

configuration is now all done!
```

3) sudo ./dc.sh up -d
4) Waited for successful start...
5) Attached to backup-container via sudo ./backup.sh attach
6) rcb status
